apps/src/studio/StudioSpriteSheet.js

```diff
@@ -26,59 +26,125 @@ var utils = require('../utils');
  *
  * @constructor
  * @param {!Object} options
- * @param {!string} options.image - URL of the sprite sheet asset.
- * @param {number} [options.width] - frame width in original asset.  Default 50.
- * @param {number} [options.height] - frame height in original asset. Default 50.
+ * @param {!string} options.assetPath - URL of the sprite sheet asset.
+ * @param {Object} [options.animations] - Array of animation descriptions,
+ *        each containing an Object: {string}type, {number}count, {number}frames
```

Comment:
Another JSDoc typedef opportunity.  Above this you could do:

```javascript
/**
 * @typedef AnimationDescription
 * @property {string} type
 * @property {number} count
 * @property {number} frames
 */
```

Then the type for this parameter would be:
```javascript
 * @param {AnimationDescription[]} [options.animations] - Array of animation descriptions
```